external/harvestPackages/harvestPackages.depproj

```diff
@@ -4,143 +4,167 @@
     <PackageReference Include="System.Security.AccessControl">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Security.Cryptography.Cng">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Security.Cryptography.OpenSsl">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Security.Cryptography.Pkcs">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Security.Cryptography.ProtectedData">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
-    <PackageReference Include="System.Security.Cryptography.X509Certificates">
-     <Version>4.3.0</Version>
+    <PackageReference Include="System.Security.Cryptography.Xml">
+      <Version>4.4.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.Security.Permissions">
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Security.Principal.Windows">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.ServiceProcess.ServiceController">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Text.Encoding.CodePages">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Text.Encodings.Web">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Threading.AccessControl">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Threading.Tasks.Dataflow">
-     <Version>4.7.0</Version>
+      <Version>4.8.0</Version>
     </PackageReference>
     <PackageReference Include="System.Threading.Tasks.Extensions">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.ValueTuple">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
   </ItemGroup>
 
+  <Import Project="$(ProjectDir)pkg/baseline/baseline.props" />
+
+  <!--
+    Custom target to update the stable versions. Ideally we would just run this target as part of the build but
+    currently the restore step happens in CLI which doesn't have matching dependencies for the package task. Currently
+    they have different versions of nuget libraries. We can look at fixing that in the future but for now this can be
+    ran once and the values can be copy/pasted into this project to update as needed.
+  -->
+  <Target Name="OutputLatestStablePackages">
+    <ItemGroup>
+      <PackageReference>
```

Comment:
Per my [previous comment](https://github.com/dotnet/corefx/pull/22704#discussion_r130117611) I don't think this is needed.